### PR TITLE
Migrate Query Workbench to 7.10

### PR DESCRIPTION
**Pull request by davidcui-amzn**
*Issue #, if available:*
N/A
*Description of changes:*
Migrate query workbench to 7.10. Tested by running both `SQL` and `PPL` queries and testing features such as `Explain`, `Download`, and sorting on columns. 

Related PR to update workflows: #844 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: workbench/kibana.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "id": "opendistroQueryWorkbench",
```
Comment (joshuali925) [inline workbench/kibana.json:2]: in odfe 1.12 workbench plugin changed to `opendistroQueryWorkbenchKibana`, could you update here?

**Review by joshuali925 — COMMENTED** [review]
Github action failed, could you update node version?
Also artifact file name changed in 7.10, needs to rename it in github actions. You can use reporting as a reference https://github.com/opendistro-for-elasticsearch/kibana-reports/blob/dev/.github/workflows/kibana-reports-test-and-build-workflow.yml#L49

File: workbench/kibana.json (hunk 1 above)
Comment (davidcui-amzn) [reply workbench/kibana.json:2]: Yes, will do

**Comment by chloe-zh** [comment]
`error opendistro-query-workbench@1.12.0.0: The engine "node" is incompatible with this module. Expected version "10.22.1". Got "10.22.0"`
Looks like the node version does not match in the workbench workflows, could you also change the node version in `.github/workflows/sql-test-and-build-workflow.yml` and `.github/workflows/sql-workbench-release-workflow.yml`?

**Review by chloe-zh — APPROVED** [review]
LGTM, thanks for the changes!

**Review by joshuali925 — APPROVED** [review]
Thanks

**Pull request merged**